src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```diff
@@ -597,4 +611,18 @@ public function theOrderShouldHavePaymentState(OrderInterface $order, $orderPaym
             sprintf('Cannot find order with "%s" order payment state in the list.', $orderPaymentState)
         );
     }
+
+    /**
+     * @Then /^there should be(?:| only) (\d+) payment(?:|s)$/
```

Comment:
These regular expressions I wrote above means the same, `?` matches 0 or 1 occurence of the token before :)